src/content/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-12-8-0.mdx

```diff
@@ -0,0 +1,53 @@
+---
+subject: Node.js agent
+releaseDate: '2024-11-19'
+version: 12.8.0
+downloadLink: 'https://www.npmjs.com/package/newrelic'
+security: []
+bugs: ["Corrected expected shape of user input in get Prompt()"]
+features: ["Added new configuration options to add custom tags (labels) to logs","Moved `entity.guid`, `entity.name`, `entity.type`, and `hostname` to `common.attributes` on logs payload instead of in every log message","Added `labels.enabled` and `labels.exclude` to config defaults","Added support for Bedrock Claude 3.5 invoke model user request"]
+---
+
+## Notes
+
+#### Features
+
+* Added `labels.enabled` and `labels.exclude` to config defaults ([#2727](https://github.com/newrelic/node-newrelic/pull/2727)) ([36adb13](https://github.com/newrelic/node-newrelic/commit/36adb13caad41a5df5a5fa9345b9dcb9061db969))
+* Added new configuration options to add custom tags (labels) to logs ([#2743](https://github.com/newrelic/node-newrelic/pull/2743)) ([d20c874](https://github.com/newrelic/node-newrelic/commit/d20c874638ebb19f8888003d4402bce17e56360f))
+    * With custom tags on logs, platform engineers can easily filter, search, and correlate log data for faster and more efficient troubleshooting, improved performance, and optimized resource utilization. To learn more about this feature see the [documentation](https://docs.newrelic.com/docs/logs/logs-context/custom-tags-agent-forwarder-logs/)

```

Comment:
```suggestion
* Added new configuration options to add custom tags (labels) to logs ([#2743](https://github.com/newrelic/node-newrelic/pull/2743)) ([d20c874](https://github.com/newrelic/node-newrelic/commit/d20c874638ebb19f8888003d4402bce17e56360f))
    * With custom tags on logs, platform engineers can easily filter, search, and correlate log data for faster and more efficient troubleshooting, improved performance, and optimized resource utilization. To learn more about this feature see the [documentation](https://docs.newrelic.com/docs/logs/logs-context/custom-tags-agent-forwarder-logs/)
    * To enable set `config.application_logging.forwarding.labels.enabled` to `true` or the environment variable `NEW_RELIC_APPLICATION_LOGGING_FORWARDING_LABELS_ENABLED` to true ([#2727](https://github.com/newrelic/node-newrelic/pull/2727)) ([36adb13](https://github.com/newrelic/node-newreli       c/commit/36adb13caad41a5df5a5fa9345b9dcb9061db969))
    * To exclude labels that were set on application from logs use `config.application_logging.forwarding.labels.exclude` or the environment variable `NEW_RELIC_APPLICATION_LOGGING_FORWARDING_LABELS_EXCLUDE`
```

Reply:
to whomever reviews this, can you please accept this, we had to tweak our changelog and this was the update